Emmanuel-Nkurunziza/My_Diary
# ft(ch1-welcoming-page): welcoming user to the UI

#### What does this PR do?
- it adds sign up/register/subscribe buttons to the welcoming page and sets it
- it adds some welcoming text and pictures to explain how and why the services My_Diary app offers are going to impact positively users life
It also adds the PULL_REQUEST_TEMPLATE

#### Description of Task to be completed
- upon click the sign up/register/subscribe buttons, a convinced visitor lands on the sign up page to register and become the user.

#### How should this be manually tested?
- clone the repo and switch to the branch ft-ch1-welcoming-page-169109257
- use the url ` https://emmanuel-nkurunziza.github.io/My_Diary/UI/html/index.html ` to open the Welcoming page
- you should successfully see the above-mentioned features of the index.html

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#169109257]( https://www.pivotaltracker.com/story/show/169109257)

#### Screenshots (if appropriate)
image of the index or welcoming page
![image](https://user-images.githubusercontent.com/52543344/66742264-90d66780-ee77-11e9-8e08-26f558a7f9c7.png)
![image](https://user-images.githubusercontent.com/52543344/66742330-b499ad80-ee77-11e9-862e-de017dc086e3.png)
